lib/core/src/lib/viewer/components/img-viewer.component.spec.ts

```diff
@@ -132,7 +132,7 @@ describe('Test Img viewer component ', () => {
             }).toThrow(new Error('Attribute urlFile or blobFile is required'));
         });
 
-        it('If  url is passed should not thrown an error', () => {
+        it('If url is passed should not thrown an error', () => {
```

Comment:
every test case should start with 'should' word, could you please correct this title?

Reply:
I only removed redundant space from it, in those test are many thing to change/improve, I don't think it should be part of this commit